docs/docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -65,7 +65,7 @@ spec:
   source:
     repoURL: ghcr.io/akuity/kargo-charts
     chart: kargo
-    targetRevision: 1.2.0
+    targetRevision: <desired version of Kargo>
```

Comment:
```suggestion
    targetRevision: <desired version of Kargo>
```

If someone were to directly copy and apply the manifest, it wouldn't work. Do you think it's worth adding a note about setting the desired `targetRevision` field before applying the manifest?

Reply:
I think angled brackets are a hard-to-miss visual cue and if someone does copy this verbatim, the mistake will be very easily discovered. It's also a relatively adroit user who would even be looking to GitOps their installation of Kargo in the first place. This won't confuse anyone.